Rebuild-Black-Business/RBB-Website
# should be single link

# Describe your PR
Business result card, "report or update" should be a single mailto: link with underline

Related to #
Fixes #

## Pages/Interfaces that will change

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

![image](https://user-images.githubusercontent.com/8475305/84327245-d9e8d380-ab4c-11ea-8e62-783493c81511.png)

## Steps to test

1. Go to /businesses
2. Make sure its a single link for the report or update link in the cards

### Additional notes
